### PR TITLE
task(settings): Fix linter warnings

### PR DESCRIPTION
**Pull request by dschom**
## Because

- While working on some NX stuff, I noticed these warnings were an issue.
- We are going to fail linting in CI if there are warnings

## This pull request

- Corrects linter warnings

## Issue that this pull request solves

Closes: # (issue number)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx [hunk 1]
```diff
@@ -82,4 +82,5 @@ export const Introduction = () => (
   </Page>
 );
 
-export default () => Introduction();
+export const introduction = () => Introduction();
```
Comment (pdehaan) [inline packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx:85]: Q: should this be exported too?
Doesn't seem consistent with above patterns and i dont think it was exported previously as a named export.

Comment (dschom) [reply packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx:85]: The linter error was essentially that the default export had no name.  IMO it's fine to export by name as well as being exported as the default.

Comment (pdehaan) [reply packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx:85]: Ah, apologies. I didn't realize that this was just a storybook file. :yolo:

Comment (chenba) [reply packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx:85]: VS Code does not warn about the lint error.  I presume it's the same for the author of that line.  Should we add fxa-setting linting to the pre-commit hook then?

Comment (dschom) [reply packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx:85]: This has been added to the husky config for lint-staged.

**Review by LZoog — APPROVED** [review]

**Comment by dschom** [comment]
This ended up being taken care of in https://github.com/mozilla/fxa/pull/15698

**Pull request closed without merging**